crates/inspection-service/Cargo.toml

```diff
@@ -8,6 +8,7 @@ homepage = "https://aptoslabs.com"
 license = "Apache-2.0"
 publish = false
 edition = "2018"
+build = "build.rs"
```

Comment:
We don't need anything from the inspection service. If we wanted to do this, I'd propose we actually add it to `aptos-node`. But, the entire telemetry system is likely to be overhauled, so let's only add it to the CLI for now.